Gopkg.toml

```diff
@@ -0,0 +1,31 @@
+# Gopkg.toml example
```

Comment:
Let's decide on if we'd rather use Deps or just use Go mods and have the mod file keep track of all the dependencies?